presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -5,11 +5,31 @@ Data Types
 Presto currently supports a limited set of data types.
 These types can be used with the standard ``CAST`` operator.
 
+.. note:: Not every of this type is supported by every connector, this is only list of types supported by Presto.
```

Comment:
"Some connectors may not support all types."

This of course raises the question of which types are supported by which connectors.